setup.py

```diff
@@ -39,7 +39,9 @@ def get_version(module='spyder_kernels'):
                 'pyzmq>=17',
                 'jupyter-client>=5.2.3',
                 'cloudpickle',
-                'wurlitzer;platform_system!="Windows"']
+                'wurlitzer;platform_system!="Windows"',
+                'mock',
+                'matplotlib']
```

Comment:
These are not package requirements, but test ones. So what you need to do is add an `extras_require` kwarg to `setup` below and list all our test requirements there (not just these two).

Our test requirements are listed here:

https://github.com/spyder-ide/spyder-kernels/blob/master/requirements/tests.txt